src/main/proto/org/dependencytrack/policy/v1/policy.proto

```diff
@@ -90,7 +90,7 @@ message Project {
   string name = 3;
   optional string version = 4;
   optional string classifier = 5;
-  bool is_active = 6;
+  optional google.protobuf.Timestamp inactive_since = 6;
```

Comment:
See comment in `CelPolicyProjectRowMapper.java`. We should avoid this breaking change.